docs/ssh-keys/deploy-ssh-key.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Deploy SSH Key
+intro: Deploy your SSH Keys to each of the servers you want the keys on.
+links:
+    overview:
+    quickstart:
+    previous:
+        - /docs/ssh-keys/link-ssh-key
+    next:
+        - /docs/servers/ssh-into-server
+    guides:
+    related:
+    featured:
+required_permissions:
+    - application:deploy
```

Comment:
```suggestion
    - ssh-key:pipeline_run
```

SSH Key does not require application permissions; and instead of deploy, the API requires the user to be able to run pipelines.

Could you update this section?